apps/backend/src/pies/domain/pie.ts

```diff
@@ -0,0 +1,28 @@
+import BigNumber from 'bignumber.js';
```

Comment:
Is it possible for us to drop bignumber.js? Ethers uses BN.js under the hood and node since v10 altready has builtin support for BigInt so in order to not repeating the same error of mixing up bignumber libraries I probably avoid this